plasmapy/particles/particle_class.py

```diff
@@ -466,6 +484,26 @@ def __str__(self) -> str:
         """Return the particle's symbol."""
         return self.particle
 
+    def particle_dict(self) -> dict:
+        """
+        Return a dictionary representation of the minimum description of a particle.
+
+        Examples
+        --------
+        >>> lead = Particle('lead')
+        >>> lead.particle_dict()
+        {'plasmapy_particle': {'type': 'Particle', 'description': {'argument': 'Pb'}}}
+        >>> electron = Particle('e-')
+        >>> electron.particle_dict()
+        {'plasmapy_particle': {'type': 'Particle', 'description': {'argument': 'e-'}}}
+
+        """
+        particle_dictionary = super().particle_dict
+        particle_dictionary["plasmapy_particle"]["description"] = {
+            "argument": self.particle
+        }
```

Comment:
```suggestion
        particle_dictionary["plasmapy_particle"]["init_params"] = {
            "argument": self.particle
        }
```

Reply:
`"init_params"` seems to be the more suitable key. I shall make this change.